adaptors/collections.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Collections Adaptor
```

Comment:
Wondering if we can somehow tag `collections` and `common` as special OpenFn adaptors?

Reply:
Maybe. Collections is definitely special. Common is only sort of special :thinking: 

So I guess to answer this I'd have to ask what we mean by "special" and what exactly we want to flag